projects/sheetmapper/README.md

```diff
@@ -0,0 +1,164 @@
+
+{ image of template spreadsheet }
+
+### Step 2. Publish your Google Sheet
+
+Follow these steps to publish a Google Sheet and get a direct link to download it as a CSV file.
+
+1. In your Google Sheet, go to **File > Share > Publish to web**.
+2. In the **Publish to the web** dialog:
+   - Select the sheet you want to share.
+   - Under **Link**, choose **CSV** as the format.
+3. Click **Publish** and confirm that you want to publish the document.
+4. A URL will be generated. Copy it. You will need this URL to configure sheetmapper.
+
+Your CSV download URL will look like this:
+```
+https://docs.google.com/spreadsheets/d/1dLkr70tfAL_-U6ipad9xLF39lHmU8k-xd08uKHkLD5M/gviz/tq?tqx=out:csv&sheet=Sheet1
+```
+You can confirm that it is working by trying it out in a web browser, it should trigger a CSV file download including the data from your sheet.
+
+**Important Notes**
+- Publishing to the web makes the document publicly accessible to anyone with the link. Use caution with sensitive information.
+- If you update the Google Sheet, the CSV link will reflect those changes automatically.
+
+### Step 3. Preview Sheetmapper locally
+
+Follow these steps to download the Sheetmapper HTML file from GitHub, open it in Visual Studio Code, and view it in your browser.
+
+#### 1: Find and Download the HTML File from GitHub
+1. Go to the [Sheetmapper HTML file on github](https://github.com/mapbox/public-tools-and-demos/)
+2. Click on the **Raw** button to view the raw HTML code.
+3. Right-click on the page and select **Save as…** to save the HTML file to your computer.
+
+#### 2: Open the HTML File in Visual Studio Code
+1. Open **Visual Studio Code** on your computer.
+2. Use **File > Open File…** or **File > Open Folder…** to open the Sheetmapper HTML file or the folder containing the Sheetmapper HTML file.
+3. Once the file is open, you should see the HTML code in Visual Studio Code.
+
+#### 3: Install the Live Server Extension
+To make it easier to preview changes in real-time, you can install the Live Server extension in Visual Studio Code:
+1. Go to the **Extensions** tab in Visual Studio Code (the square icon on the sidebar).
+2. Search for **Live Server** and click **Install**.
+3. Once installed, you’ll be able to start a live server to preview the HTML file.
+
+#### 4: Run the HTML File in Your Browser
+1. With the HTML file open, right-click in the editor window and select **Open with Live Server**.
+2. Your default browser will open, displaying the HTML page. Any changes you make and save will automatically refresh in the browser. You will see a browser alert with an error if you have not yet configured Sheetmapper.
+
+## Troubleshooting Tips
+- Ensure that **Live Server** is properly installed and enabled if using the live preview.
+- If the file does not open in your browser, check the file path or try opening it manually by dragging it into your browser window.
+
+### Step 4. Configure Sheetmapper
```

Comment:
I think this step should be moved up to above Step 3 - INstall Live Server.  I don't see why we should run a file in the browser before it has been configured.

Reply:
Will do!